app/assets/javascripts/messages.js

```diff
@@ -123,6 +123,8 @@ In order to restore the current window, a reload is necessary.`,
   "auth.reset_new_password": "Please input your new password!",
   "auth.reset_new_password2": "Please repeat your new password!",
   "auth.reset_token": "Please input the token!",
+  "auth.reset_token_not_supplied":
+    "There was no token was found in the URL. Check your E-Mails to get the correct URL.",
```

Comment:
`There was no token found in the URL. ...`

Reply:
:smile: